development-docs/TESTING.md

```diff
@@ -148,29 +148,36 @@ The following table shows currently used tags:
 | loadbalancer       | Execute tests which use external lister of type loadbalancer                       |
-| bridge             | Execute tests which use Kafka Bridge                                               |
-| specific           | Specific tests, which cannot be easily added to other categories                   |
 | networkpolicies    | Execute tests which use Kafka with Network Policies                                |
-| tracing            | Execute tests for Tracing                                                          |
 | prometheus         | Execute tests for Kafka with Prometheus                                            |
-| oauth              | Execute tests which use OAuth                                                      |
+| tracing            | Execute tests for Tracing                                                          |
 | helm               | Execute tests which use Helm for deploy cluster operator                           |
-| olm                | Execute tests which use OLM for deploy cluster operator                            |
+| oauth              | Execute tests which use OAuth                                                      |
+| recovery           | Execute recovery tests                                                             |
 | connectoroperator  | Execute tests which deploy KafkaConnector resource                                 |
 | connect            | Execute tests which deploy KafkaConnect resource                                   |
 | connects2i         | Execute tests which deploy KafkaConnectS2I resource                                |
 | mirrormaker        | Execute tests which deploy KafkaMirrorMaker resource                               |
 | mirrormaker2       | Execute tests which deploy KafkaMirrorMaker2 resource                              |
 | conneccomponents   | Execute tests which deploy KafkaConnect, KafkaConnectS2I, KafkaMirrorMaker2, KafkaConnector resources |
+| bridge             | Execute tests which use Kafka Bridge                                               |
 | internalclients    | Execute tests which use internal (from pod) kafka clients in tests                 |
 | externalclients    | Execute tests which use external (from code) kafka clients in tests                |
+| olm                | Execute tests which use OLM for deploy cluster operator                            |
+| metrics            | Execute tests where metrics are used                                               |
+| cruisecontrol      | Execute tests which deploy CruiseControl resource                                  |
+| scale              | Execute tests where are resources (its pods) scaled up/down                        |
+| rollingupdate      | Execute tests where is rolling update triggered                                    |
+| norollingupdate    | Execute tests where resource (and its pods) should not roll                        |
```

Comment:
Well....you should use only one tag and put it to every test which tests rolling update somehow from my POV.

Reply:
Yep that's sound fair. I was thinking that if you use the `norollingupdate` tag when someone add some feature and you want to try, if the tests where rolling update should not be triggered are passing, you use this one to skip all other rolling update tests.